### PR TITLE
declare function-specific variables with local

### LINKED ISSUE #16838: Validation workflow need to be updated in ubuntu.md
The current scripts did not print `Cluster validation succeeded`:

```

Kubernetes cluster is running.  The master is running at:

  http://10.10.102.153:8080

Using master 10.10.102.153
It looks as if you don't have a compiled kubectl binary

If you are running from a clone of the git repo, please run
'./build/run.sh hack/build-cross.sh'. Note that this requires having
Docker installed.

If you are running from a binary release tarball, something is wrong. 
Look at http://kubernetes.io/ for information on how to contact the 
development team for help.

```
I guess some validation steps are actually un-needed in ubuntu workflow? maybe we need to remove that line in the doc (at least)? @dalanlan

### PR DESCRIPTION
**Pull request by resouer**
Declare function-specific variables with local, for consistency and follow [codestyle](https://google.github.io/styleguide/shell.xml)

@dalanlan You can review this, please.

### CODE REVIEW

File: cluster/ubuntu/util.sh [hunk 1]
```diff
@@ -127,6 +127,7 @@ function verify-cluster {
     ((ii=ii+1))
   done
 
+  echo "Cluster validation succeeded"
```
Comment (dalanlan) [inline cluster/ubuntu/util.sh]: -1 for this.

Comment (dalanlan) [inline cluster/ubuntu/util.sh]: Ref #16838 for more detail

**Comment by dalanlan** [comment]
I'm fine with the second commit. Yet this pr doesn't close #16838, i'm afraid.

**Comment by resouer** [comment]
So jump to #16850  This PR has no relationship with #16838 now

**Comment by resouer** [comment]
@eparis  rebase done, just minor fix. Feel free to merge it in.

**Pull request merged**